AddOns/BasicFinancials/app/src/page/AssistedSetupBF.Page.al

```diff
@@ -0,0 +1,154 @@
+            {
+                ApplicationArea = Basic, Suite;
+                Caption = ' ';
+                ShowCaption = false;
+                Editable = false;
+                ToolTip = 'For more information, see the documentation.';
+            }
+            field(HelpLink; 'Basic documentation')
+            {
+                ApplicationArea = Basic, Suite;
+                Caption = ' ';
+                ShowCaption = false;
+                Editable = false;
+                ToolTip = 'Basic documentation';
+
+                trigger OnDrillDown()
+                begin
+                    Hyperlink('https://go.microsoft.com/fwlink/?linkid=');
+                end;
+            }
+            field(ConsentPart; 'To enable the service you must read and accept the terms of use.')
+            {
+                ApplicationArea = Basic, Suite;
+                Caption = ' ';
+                ShowCaption = false;
+                Editable = false;
+                ToolTip = 'By enabling this extension you accept the terms of use. To enable the service you must read and accept the terms of use.';
+            }
+            field(TermsOfUseLink; 'Basic terms of use')
+            {
+                ApplicationArea = Basic, Suite;
+                Caption = ' ';
+                ShowCaption = false;
+                Editable = false;
+                ToolTip = 'Basic terms of use';
+
+                trigger OnDrillDown()
+                begin
+                    Hyperlink('https://go.microsoft.com/fwlink/?linkid=');
+                end;
+            }
+            field(AcceptConsent; ConsentAccepted)
+            {
+                ApplicationArea = Basic, Suite;
+                Editable = true;
+                Caption = 'I understand and accept the terms';
+                ToolTip = 'Acknowledge that you have read and accept the terms.';
+            }
+            /* Temporarily removed due to issue regarding License check
+            field(IsSupportedLicenses; IsSupportedLicenses)
+            {
+                ApplicationArea = Basic, Suite;
+                Editable = true;
+                Caption = 'Basic license has been assigned';
+                ToolTip = 'To complete the Basic setup the Basic license must be assigned to at least one user.';
+                trigger OnValidate()
+                var
+                    BasicMgmt: Codeunit "Basic Mgmt BF";
+                    NotSupportedLicensesErr: Label 'At least one user must have the Basic license.';
+                begin
+                    IsSupportedLicenses := BasicMgmt.IsSupportedLicense();
+                    if not IsSupportedLicenses then
+                        Error(NotSupportedLicensesErr);
+                end;
+            }
+            */
+            field(IsSupportedCompanies; IsSupportedCompanies)
+            {
+                ApplicationArea = Basic, Suite;
+                Editable = true;
```

Comment:
It work as designed. If the user try to changes it and there still too many companies, a message will appear. If the user has removed enough companies, the value vil be changes to true.

If you think it is better to block a change and force the user to close and re-open the page, so it is fine by me.. It will require a very small coding changes.

Reply:
I discussed this with Fredrik and Brian and we said that it should be only notification rather than blocker. Lets remove this field and provide in the page caption that this extension should work only for one company. Don't block setup if there are mpore than 1.

Reply:
I understand the notification and blocker differently. I have removed all blocking from OnInstall af the extension, this meens that the Extension can be install without being blocked. My understanden was that we aggreed on that the assisted setup should be understood as a notification. By blocken the assisted setup, it is possible for us partners to see if the prerequisites have been fulfilled at the time of setup. This valuable knowledge is lost if we allow the setup to be completed. I do not see that the solution you propose gives value to the customer or Microsoft. But, of course, the choice is entirely yours.

Please confirm, that I shall remove the "1 company"-blocking from the assisted setup.

Reply:
![image](https://user-images.githubusercontent.com/47662057/86051872-6bfe4080-ba56-11ea-9686-2441c248953b.png)

@PredragMaricic I have tried to adjust the assisted setup based on extension-advanced-example. I could use your guide, I don't quite understand what you mean by "provide in the page caption" do you expect that the text only will be shown if there is currently more than one company. I can't / don't know how to control the page caption by a variable
Secondly, I have an action button in the lower left corner that I can't get away, can you give me a clue to find the error.

https://docs.microsoft.com/en-us/dynamics365/business-central/dev-itpro/developer/devenv-extension-advanced-example

Reply:
New version with Notification.

![image](https://user-images.githubusercontent.com/47662057/86226820-7d496900-bb8c-11ea-8658-ac39a3badd99.png)